setup/steps/enable_pgss.go

```diff
@@ -0,0 +1,56 @@
+package steps
+
+import (
+	"errors"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	s "github.com/pganalyze/collector/setup/state"
+	"github.com/pganalyze/collector/setup/util"
+)
+
+var EnablePgss = &s.Step{
+	Description: "Enable pg_stat_statements",
+	Check: func(state *s.SetupState) (bool, error) {
+		spl, err := util.GetPendingSharedPreloadLibraries(state.QueryRunner)
+		if err != nil {
+			return false, err
+		}
+
+		return strings.Contains(spl, "pg_stat_statements"), nil
+	},
+	Run: func(state *s.SetupState) error {
+		var doAdd bool
+		if state.Inputs.Scripted {
+			if !state.Inputs.EnablePgStatStatements.Valid || !state.Inputs.EnablePgStatStatements.Bool {
+				return errors.New("enable_pg_stat_statements flag not set but pg_stat_statements not in shared_preload_libraries")
+			}
+			doAdd = state.Inputs.EnablePgStatStatements.Bool
+		} else {
+			err := survey.AskOne(&survey.Confirm{
+				Message: "Add pg_stat_statements to shared_preload_libraries (will be saved to Postgres)?",
```

Comment:
`Add pg_stat_statements to shared_preload_libraries (will be saved to Postgres - requires a restart)?`